### PR TITLE
Update groups of experts

### PR DESCRIPTION
**Pull request by Mitso**

### CODE REVIEW

**Comment by Mitso** [comment]
Request: Could I ask for one change? Expert should be "Tune Me Expert" as this would generally be someone separate to a moderator, e.g. an external person coming in

Please review 
screen shot 2017-10-30 at 3 18 52 pm

File: tuneme/templates/new/comments/comment.html [hunk 1]
```diff
@@ -21,7 +25,7 @@ <h4 class="comments__alias-author">
 
 {% if node.is_removed %}
     <h6 class="comments__description">
-      {% trans "This comment has been removed by the community. Please read our platform rules." %}
+      {% trans "This comment has been removed b ~~ y the community. Please read our platform rules." %}
```
Comment (projectmushroom) [inline tuneme/templates/new/comments/comment.html]: ~~ probably not intended

**Review by projectmushroom — APPROVED** [review]
👍🍰 just a small comment

File: tuneme/templates/new/comments/comment.html (hunk 1 above)
Comment (Mitso) [reply tuneme/templates/new/comments/comment.html]: Thanks @projectmushroom!

**Comment by smarzban** [comment]
:cake:

**Comment by johan** [comment]
@Mitso, try to avoid @ signs, or use real github usernames like @saeedmb8 and @projectmushroom, so you don't email random other people on github. The web github comment field has great sane autocomplete suggestions.

**Pull request merged**